packages/react-components/src/components/VideoGallery/VideoGallery.tsx

```diff
@@ -105,6 +114,49 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
     return !!(layout === 'floatingLocalVideo' && remoteParticipants && remoteParticipants.length > 0);
```

Comment:
is there a reason this is a callback and not just a const variable? (with useMemo if we need the memoization optimization)